lib/experimental/new_embed.dart

```diff
@@ -169,7 +169,26 @@ abstract class TabView {
 }
 
 class EditorTabView extends TabView {
-  const EditorTabView(DElement element) : super(element);
+  EditorTabView(DElement element, EditorFactory editorFactory)
```

Comment:
I actually hadn't intended the EditorTabView to be used for anything more than showing or hiding what was inside it. I should have wrapped the textarea in a div to make that more clear in the original code.

What do you think about changing the HTML to:

```
<section id="tab-container" class="flex-auto d-flex flex-column p-1">
	<div class="tabview flex-auto" id="editor-view" selected>
		<div id="editor" spellcheck="false"></div>
	</div>
    <textarea class="tabview flex-auto" id="test-view" spellcheck="false"></textarea>
    <div class="tabview flex-auto" id="console-view"></div>
</section>
```

And separating the concerns of showing and hiding the editor tab content from everything else the editor does?

Reply:
I think we can clean up everything (both HTML and Dart) in a follow up PR once this initial integration actually works. Small steps ;-)